### DISCUSSION TITLE
luarocks is not executable

### DISCUSSION THREAD

**Discussion by appicon007** [mason-org/mason.nvim#1630] [Q&A]
Hey,
tried to install luarocks formatter and came up with the following error. I can not find a solution so far... maybe someone can help please.

OS: Windows 10

```
Failed
    ◍ luaformatter
      ▼ Displaying full log
        Installing luarocks package luaformatter@scm-1…
        spawn: luarocks failed with exit code - and signal -. luarocks is not executable
```

**Comment by ryuheechul** [comment]
_disclaimer, I never used luarocks just yet_ but you probably need to install [luarocks](https://github.com/luarocks/luarocks/wiki/Download#installing) first .
